src/EventSubscriber/HeaderResponseSubscriber.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+
+class HeaderResponseSubscriber implements EventSubscriberInterface
+{
+    private string $apiVersion;
+
+    public function __construct(string $apiVersion)
+    {
+        $this->apiVersion = $apiVersion;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            ResponseEvent::class => 'addVersionHeader',
+        ];
+    }
+
+    public function addVersionHeader(ResponseEvent $event): void
+    {
+        $route = explode('_', $event->getRequest()->attributes->get('_route'));
```

Comment:
Fatal error may occur here since `$event->getRequest()->attributes->get('_route')` can be null